src/main/java/core/basesyntax/service/FileReaderCsv.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileReaderCsv {
```

Comment:
Don't get attached to specific types
```suggestion
public interface Reader {
```